jwst/associations/lib/dms_base.py

```diff
@@ -74,23 +74,23 @@
 
 # Coronographic exposures
 CORON_EXP_TYPES = [
-    'mir_lyot',
     'mir_4qpm',
+    'mir_lyot',
     'nrc_coron'
 ]
 
 # Exposures that get Level2b processing
 IMAGE2_SCIENCE_EXP_TYPES = [
+    'fgs_image',
+    'mir_4qpm',
     'mir_image',
     'mir_lyot',
-    'mir_4qpm',
     'nis_ami',
     'nis_image',
-    'nrc_image',
     'nrc_coron',
-    'nrc_tsimage',
+    'nrc_image',
     'nrs_mimf',
-    'fgs_image',
+    'nrc_tsimage',
```

Comment:
If you're trying to achieve strict alphabetical ordering of these elements, `nrc_tsimage` needs to be above `nrs_mimf` in the list ;-)